components/modules/Kconfig

```
@@ -74,6 +74,36 @@ config LUA_MODULE_CAN
   help
       Includes the can module.
 
+config LUA_MODULE_CRYPTO
+  bool "Crypto module"
+  default "n"
+  help
+      Includes the crypto module.
+
+menu "Crypto module hashing algorithms"
```

Cosmetic: Hide this menu when the crypto module is not selected:
```suggestion
menu "Crypto module hashing algorithms"
  depends on LUA_MODULE_CRYPTO

```